### PR TITLE
Add GetTotalBondedDelegation() to Term

### PR DESCRIPTION
**Pull request by eunsoo-icon**
- Add rrep to Term
- Add 'getNetworkValue' JSON-RPC API for testing
- Fix Term.newNextTerm() to read irep, rrep and period from networkValue

### CODE REVIEW

**Comment by eunsoo-icon** [comment]
* remove totalBondedDelegation field
* provide TotalBondedDelegationI() method only

File: icon/iiss/icstate/term.go [hunk 1]
```diff
@@ -285,21 +324,25 @@ func (term *Term) ToJSON() map[string]interface{} {
...
 	jso["irep"] = term.irep
+	jso["rrep"] = term.rrep
 	jso["period"] = term.period
 	jso["preps"] = term.prepSnapshots.toJSON()
 
 	return jso
 }
 
-func (term *Term) NewNextTerm(totalSupply *big.Int, totalDelegated *big.Int) (*Term, error) {
+func (term *Term) NewNextTerm(s *State, totalSupply *big.Int, totalDelegated *big.Int) (*Term, error) {
```
Comment (eunsoo-icon) [inline icon/iiss/icstate/term.go:336]: move to ExtentionStateImpl

File: icon/iiss/icstate/term.go [hunk 2]
```diff
@@ -13,8 +13,10 @@ import (
 )
 
 type PRepSnapshot struct {
-	owner     module.Address
-	delegated *big.Int
+	owner            module.Address
+	delegated        *big.Int
+	bonded           *big.Int
```
Comment (eunsoo-icon) [inline icon/iiss/icstate/term.go:18]: remove

File: icon/iiss/icstate/term.go (hunk 1 above)
Comment (eunsoo-icon) [reply icon/iiss/icstate/term.go:336]: Do not move to ExtentionStateImpl

Make new constructor
```go
func NewNextTerm(term *Term, irep *big.int...)
```

**Review by goldworm-icon — APPROVED** [review]

**Pull request merged**